### ISSUE TITLE
I cannot the reproduce nyu performance with densenet161 model

### ISSUE THREAD

**Issue by songya** [cogaplex-bts/bts#111]
I used the default parameter settings using the file named "argument_train_nyu.txt (densenet161_bts setting).
I trained three times but the d1 performance  had never reached to the accuracy you published.
According to your paper, it should achieve d1 = 0.885 but my results are 0.875, 0.877, and 0.878.
The only difference is a multiprocessing option. 
With 'on-line eval' setting, the multiprocessing option cannot work. 
I used lr=1e-4, weight decay = 1e-2, adam_eps = 1e-3.
Could you explain why I cannot reproduce your result?

**Comment by freshjh** [comment]
The performance reported in the paper is produced by a big batchsize, 16.  You can refer to the section of **implementation detail** for  more information.
